ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -298,6 +313,9 @@ Item {
                             if (!root.canPost) {
                                 return qsTr("Sorry, you don't have permissions to post in this channel.")
                             }
+                            if (d.sending) {
+                                return qsTr("Sending in progress...")
```

Comment:
Maybe just `Sending..`? It's more common I'd say. 😄

Reply:
I _think_ we even have some designs for this somewhere